prompt.md

```diff
@@ -87,7 +87,7 @@ Description of intent: An image processing job which requires medium amount of c
 }
 
 
-Description of intent: A simple job that just sleep 30 seconds and is only allowed to run in us-central1 region.
+Description of intent: A simple job that just sleep 30 seconds and is only allowed to run in us-central1 region. However we want the VMs to be reliable instead of Preemptible.
```

Comment:
Curious that if we only mention "However we want the VMs to be reliable", will it figures out that Preemptible vm should not be used.

Reply:
Yes! Gemini is able to handle this case:

https://paste.googleplex.com/5882412649414656